Caddyfile.development

```diff
@@ -1,10 +1,17 @@
-http://localhost:8000
+:8000 {
```

Comment:
Have we tested that this doesn't try and make it do SSL? Last time we had to make that clear.

Reply:
Thank you Daniel.

Reply:
thanks